src/connection.ts

```diff
@@ -115,6 +117,45 @@ export class Connection {
     })
   }
 
+  public async createStream(params: { stream: string; arguments: CreateStreamArguments }): Promise<true> {
+    this.logger.debug(`Create Stream...`)
+    const res = await this.sendAndWait<CreateStreamResponse>(new CreateStreamRequest(params))
+    if (res.code === STREAM_ALREADY_EXISTS_ERROR_CODE) {
+      return true
+    }
+    if (!res.ok) {
+      throw new Error(`Create Stream command returned error with code ${res.code}`)
+    }
+
+    this.logger.debug(`Create Stream response: ${res.ok} - with arguments: '${inspect(params.arguments)}'`)
+    return res.ok
+  }
+
+  public async deleteStream(params: { stream: string }): Promise<true> {
+    this.logger.debug(`Delete Stream...`)
+    const res = await this.sendAndWait<DeleteStreamResponse>(new DeleteStreamRequest(params.stream))
+    if (!res.ok) {
+      throw new Error(`Delete Stream command returned error with code ${res.code}`)
+    }
+    this.logger.debug(`Delete Stream response: ${res.ok} - '${inspect(params.stream)}'`)
+    return res.ok
+  }
+
+  public async queryPublisherSequence(params: { stream: string; publisherRef: string }): Promise<bigint> {
+    const res = await this.sendAndWait<QueryPublisherResponse>(new QueryPublisherRequest(params))
+
```

Comment:
Try to avoid empty lines ... (I know it0s my OCD ...)

Reply:
I'll keep in mind, removed the empty line